### ISSUE TITLE
Umbraco 13 error

### ISSUE THREAD

**Issue by lassespilling** [hfloyd/Dragonfly.Umbraco10.SchemaImporter#2]
Would it be possible to update this package to support Umbraco 13? :)

**Currently it seems it does not write package.xml to Data/TEMP folder:**
error: Failed to upload /umbraco/Data/TEMP/SchemaImporter/package-7f84b677a1.xml: 200 undefined

**Comment by chris-evansnz** [comment]
@hfloyd  I also saw this happen in a clean install of Umbraco v13.7.2 using the Dragonfly.Umbraco10.SchemaImporter version 2.2.0

I chose the package.xml file to upload on the Schema Importer dashboard, and when I clicked Upload, I saw this error appear as a notification

![Image](https://github.com/user-attachments/assets/3c96c553-dc36-4039-b8c5-0f843b5adb40)

There were two corresponding errors in the log, which mentioned a specific field by alias name.

I figured out that this was because one of the nodes in the package had a field with that alias, using a custom datatype that wasn't also included in the package. 

So presumably when it tried to create that field, it errored out because the datatype didn't exist in the new project. Once I added that custom datatype into the package, I was able to import it ok.

**Comment by hfloyd** [comment]
Yes, this package is just a very simple UI for the built-in Umbraco Core functionality from the previous versions' back-office package import. During the import it returns errors if there are schema-related reasons everything cannot be imported. The Umbraco Log will usually give enough details so you can manually edit the package file to make it importable. That's why I have the message about checking the Log on any displayed error.

**Comment by chris-evansnz** [comment]
100% - it is really handy to have this functionality back! Thanks for putting the package together.
